### PR TITLE
Update quickstart.mdx

### LINKED ISSUE #2991: [DOCS] The point can be improved by adding other sentence, where users will not create pull request in their forked repo.
### Description

- [When Following this docs](https://linkfree.eddiehub.io/docs/how-to-guides/editing)  13th point, It just says to create a Pull request but in that if the base branch is not set to `EddieHubCommunity/LinkFree `  then it will create a PR in your Forked branch.

### Screenshots

doc-update

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by shubhsharma19**
## Fixes Issue

fixes #2991 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2915"></a>

### CODE REVIEW

File: pages/docs/quickstart.mdx [hunk 1]
```diff
@@ -122,6 +122,8 @@ You can skip `optional extended message`
 | 3rd dropdown | Your repository (source)             | `your-username/LinkFree`     |
 | 4th dropdown | Your repository branch (source)      | `your-username-patch-1`      |
 
+If the Base repository (destination) from 1st dropdown is set to `your-username/LinkFree` change it to `EddieHubCommunity/LinkFree`
+
 ![compare across forls](https://user-images.githubusercontent.com/82668196/208292189-513b4213-a3a7-4a45-a834-ac6ce6b9217a.png)
```
Comment (Njong392) [inline pages/docs/quickstart.mdx:127]: While you're at it, perhaps you could fix this typo. What do you think?

```suggestion
![compare across forks](https://user-images.githubusercontent.com/82668196/208292189-513b4213-a3a7-4a45-a834-ac6ce6b9217a.png)
```

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: pages/docs/quickstart.mdx (hunk 1 above)
Comment (shubhsharma19) [reply pages/docs/quickstart.mdx]: Oh, nice thanks for this!!!

File: pages/docs/quickstart.mdx [hunk 2]
```diff
@@ -122,7 +122,9 @@ You can skip `optional extended message`
 | 3rd dropdown | Your repository (source)             | `your-username/LinkFree`     |
 | 4th dropdown | Your repository branch (source)      | `your-username-patch-1`      |
 
-![compare across forls](https://user-images.githubusercontent.com/82668196/208292189-513b4213-a3a7-4a45-a834-ac6ce6b9217a.png)
+If the Base repository (destination) from 1st dropdown is set to `your-username/LinkFree` change it to `EddieHubCommunity/LinkFree`
```
[the remark this answers is not in the document]
Comment (shubhsharma19) [reply pages/docs/quickstart.mdx]: Looks good to me 🎉

**Review by Sativarsainath-26 — APPROVED** [review]
Looks Good with the changes:+1:

**Comment by shubhsharma19** [comment]
Thanks @Sativarsainath-26

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**